### PR TITLE
[ruby] update to ruby2.2

### PR DESCRIPTION
**Pull request by fsmanuel**
@halo can you review?

### CODE REVIEW

File: cookbooks/system/ruby/recipes/install.rb [hunk 1]
```diff
@@ -1,22 +1,30 @@
-parcel 'ruby2.0'
-parcel 'ruby2.0-dev'
+execute "add-apt-repository ppa:brightbox/ruby-ng" do
```
Comment (fsmanuel) [inline cookbooks/system/ruby/recipes/install.rb]: is there a better way to add the ppa?

Comment (halo) [inline cookbooks/system/ruby/recipes/install.rb]: [Yes](stackoverflow.com/questions/9576124). I believe we already have [apt here](https://github.com/funkensturm/warehouse/tree/master/cookbooks/providers/apt).

**Comment by fsmanuel** [comment]
@halo updated!

**Comment by fsmanuel** [comment]
@halo ping

**Comment by halo** [comment]
👍

**Comment by fsmanuel** [comment]
Thanks!

**Pull request merged**